src/index.js

```diff
@@ -4,9 +4,9 @@ import { createStore } from 'redux';
 import { Provider } from 'react-redux';
 import App from './components/App';
 import rootReducer from './reducers/index';
+import './index.css';
```

Comment:
- [ ] [OPTIONAL] Instead of having a big stylesheet for various components why don't you try using CSS modules? They go pretty well with React.

[Read more on CSS modules here.](https://github.com/css-modules/css-modules)